qemu/tests/cfg/rng_hotplug.cfg

```diff
@@ -40,6 +40,10 @@
             driver_name = virtio
             check_rngd_service = "service rngd status"
             start_rngd_service = "service rngd start"
+        s390x:
```

Comment:
Should add the default rng_driver = "virtio-rng-pci" in front of this line, Yihuang has also suggested this in the .py file part.